paths/graphql/02-first-use.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Use GraphQL for the First Time
+permalink: /graphql/first-use
+next-page: /graphql/first-query
+facilitator: false
+sidebar:
+  nav: "github-graphql"
+main-content: |
+
+  When we introduced APIs, the technology that allows programs to interact with existing information, we mentioned GraphQL. In this section, we are going to use a tool that enables you to experiment with GraphQL quickly and easily.
+
+  ![img](../images/gifs/graphql/first-query.gif)
+
+  ### Navigating to GraphQL
+   1. [Navigate to the GraphQL explorer](https://developer.github.com/v4/explorer/).
+   1. Once the page has loaded, click the "Sign in with GitHub" button. If you don't have a GitHub account, [here are some instructions for getting an account setup](https://help.github.com/articles/signing-up-for-a-new-github-account/).  
+
+      After logging in, The GraphQL interface displays a sample query in the left pane that should look like this:
```

Comment:
For what it's worth, if they have accessed the Explorer in the past, this is not what they will see. They will see the last query they ran.